contracts/attesters/hydra-s1/HydraS1SimpleAttester.sol

```diff
@@ -90,11 +90,10 @@ contract HydraS1SimpleAttester is IHydraS1SimpleAttester, HydraS1Base, Attester
    * @param request users request. Claim of having an account part of a group of accounts
    * @param proofData provided to back the request. snark input and snark proof
    */
-  function _verifyRequest(Request calldata request, bytes calldata proofData)
-    internal
-    virtual
-    override
-  {
+  function _verifyRequest(
```

Comment:
Let's use the same linter/prettier as we used. This will remove a lot of diff everywhere (in all files we are not going to deploy)